uint32 id
uint32 state_id
# pose (x, y, theta)
string dev_name
geometry_msgs/Point odom_pose
geometry_msgs/Point corrected_pose
float32[] ranges